eth/polar/polaris.go

```diff
@@ -111,14 +113,37 @@ func NewWithNetworkingStack(
 		log.Root().SetHandler(logHandler)
 	}
 
-	// Build and set the RPC Backend and other services.
-	pl.backend = NewBackend(pl, stack.ExtRPCEnabled(), cfg)
-	pl.txPool = txpool.NewTxPool(host.GetTxPoolPlugin())
-	pl.miner = miner.NewMiner(pl)
+	pl.backend = NewBackend(pl, pl.stack.ExtRPCEnabled(), pl.cfg)
 
 	return pl
 }
 
+// Init initializes the Polaris struct.
+func (pl *Polaris) Init() error {
+	pl.miner = miner.New(pl)
+	// eth.miner.SetExtra(makeExtraData(config.Miner.ExtraData))
+
+	var err error
+	legacyPool := legacypool.New(
+		pl.cfg.LegacyTxPool, pl.Blockchain(),
+	)
+
+	pl.txPool, err = txpool.New(big.NewInt(0), pl.blockchain, []txpool.SubPool{legacyPool})
```

Comment:
```suggestion
	pl.txPool, err = txpool.New(new(big.Int).SetUint64(pl.cfg.LegacyTxPool.PriceLimit), pl.blockchain, []txpool.SubPool{legacyPool})
```
as done in geth:
<img width="707" alt="image" src="https://github.com/berachain/polaris/assets/113940816/ea2f673c-3b00-471e-9b18-19c8035dd863">